components/src/dynamo/vllm/args.py

```
@@ -269,35 +269,35 @@ def parse_args() -> Config:
-    )
-    has_connector_flag = args.connector is not None
+    # args.connector can never be None or an empty list, since it's defaulted to ["nixl"]
+    normalized = [c.lower() for c in args.connector]
 
-    if has_kv_transfer_config and has_connector_flag:
+    invalid = [c for c in normalized if c not in VALID_CONNECTORS]
+    if invalid:
         raise ValueError(
-            "Cannot specify both --kv-transfer-config and --connector flags"
+            f"Invalid connector(s): {', '.join(invalid)}. Valid options are: {', '.join(sorted(VALID_CONNECTORS))}"
         )
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)